lib/daily.js

```diff
@@ -0,0 +1,135 @@
+var _ = require('lodash');
+var moment = require('moment');
+var Promise = require('bluebird');
+var debug = require('debug')('daily');
+var nconf = require('nconf');
+ 
+var mongo = require('./mongo');
+
+
+function byDayStats(req) {
```

Comment:
can you please take a look to the indentation in this function? there are some inconsistencies, thanks! :oncoming_police_car: